halyard-config/src/main/java/com/netflix/spinnaker/halyard/model/v1/Updateable.java

```diff
@@ -100,4 +105,39 @@
 
     return errors;
   }
+
+  /**
+   * Validate this entire class
+   *
+   * @return The list of errors when validation fails ([] when there are no failures and validation has passed).
+   * @throws IllegalAccessException
+   * @throws NoSuchFieldException
+   */
+  default public List<String> validate() throws IllegalAccessException, NoSuchFieldException {
+    List<String> errors = new ArrayList<>();
+
+    if (this instanceof Account) {
```

Comment:
Will this only apply to `Account`s? Why is this in the interface? Seems like this ought to be in `Account`, particularly if you are planning on using `Updateable` for other things.